src/modules/personenkontext/persistence/dbiam-personenkontext.repo.spec.ts

```diff
@@ -76,6 +76,15 @@ describe('dbiam Personenkontext Repo', () => {
         });
     });
 
+    describe('findByRolle', () => {
+        it('should return all personenkontexte for a rolle', async () => {
+            const rolleUUID: string = faker.string.uuid();
+            await Promise.all([sut.save(createPersonenkontext(false, { rolleId: rolleUUID }))]);
```

Comment:
No need to use `Promise.all`, if you're only saving one entity. Just `await` the call to `sut.save`